searx/engines/ask.py

```diff
@@ -2,7 +2,7 @@
 """Ask.com"""
 
 from urllib.parse import urlencode
-import dateutil
+import dateutil.parser
```

Comment:
I've seen a few cases where the imports changed (`import x` to `import x.y`) - just want to double check they aren't accidental.

Reply:
Not sure why, but when we `import dateutil` pyright reports:

     searx/engines/ask.py:59:41 - error: "parser" is not a known \
        member of module "dateutil" (reportAttributeAccessIssue)

Reply:
Does from x import y work in these scenarios?